snp_psir_bringup/launch/psir_real.launch

```diff
@@ -1,69 +1,25 @@
 <?xml version="1.0"?>
 <launch>
 
-  <arg name="robot_ip" default="192.168.0.11"/>
+  <arg name="robot_ip" default="169.254.60.1"/>
   <arg name="robot_port" default="502"/>
 
-  <arg name="robot_name" default="psir" />
-  <!-- Enable/Disable gripper -->
-  <arg name="has_gripper" default="false" />
-
-  <!-- Decide if PSS4000 or PNoz-Multi is used -->
-  <arg name="PSS4000" default="false" />
-
- 	<!-- URDF file-->
-  <arg name="urdf_file" default="$(find snp_psir_description)/urdf/snp_psir_demo.urdf.xacro" />
-
-  <!-- Robot description argument -->
-  <arg name="robot_description" value="$(arg robot_name)_description" />
-
-  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
-  <param name="robot_description" type="str" command="$(find xacro)/xacro --inorder '$(arg urdf_file)' sim:=false"/>
-
-  <rosparam file="$(find snp_psir_bringup)/config/arm_controller.yaml" command="load" />
-  <!--node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
-    ns="/$(arg robot_name)" args="joint_state_controller joint_trajectory_controller" /-->
-
 	<!-- run the "real robot" interface nodes -->
-  <!--   - this typically includes: robot_state, motion_interface, and joint_trajectory_action nodes -->
-  <!--   - replace these calls with appropriate robot-specific calls or launch files -->
     <include file="$(find psir_description)/launch/robot.launch" >
-      <arg name="has_gripper" value="$(arg has_gripper)" />
       <arg name="load_robot_description" value="false" />
     </include>
 
     <arg name="modbus_offset" default="512" />
 
-    <!-- PSS4000 settings -->
-    <group if="$(arg PSS4000)">
```

Comment:
So here we are assuming PSS4000 is always false, right?

Reply:
Yes this is ok for our hardware setup